src/annotator/anchoring/range.js

```diff
@@ -433,13 +421,11 @@ export class SerializedRange {
    * Creates a range suitable for storage.
    *
    * root           - A root Element from which to anchor the serialization.
-   * ignoreSelector - A selector String of elements to ignore. For example
-   *                  elements injected by the annotator.
    *
    * Returns an instance of SerializedRange.
    */
-  serialize(root, ignoreSelector) {
-    return this.normalize(root).serialize(root, ignoreSelector);
+  serialize(root) {
+    return this.normalize(root).serialize(root);
```

Comment:
If this no longer needs an `ignoreSelector`, then I "think" this can now just be

`return this`

if so, then this will also remove the 3-way dependency and we can break each of this ranges objects into separate modules much easier (to-do later of course).

Reply:
`return this` would ignore the `root` argument which affects the behavior of `normalize(...)`. I'm not certain that is a safe change yet so I'd say it is out of scope for this PR.